Sources/Core/Utilities/DispatchTimer/DispatchTimer.swift

```diff
@@ -49,5 +51,11 @@ public extension DispatchTimer {
   func stop() {
     timer?.cancel()
     timer = nil
+    isExecuting = false
+  }
+  
+  var isActive: Bool {
+    guard let timer = timer else { return false }
+    return !timer.isCancelled && isExecuting
```

Comment:
Could we simplify this by only checking if the timer is not nil & not cancelled? From what I noticed, we set isExecuting to true only when we instantiate the timer.

Reply:
I've found out, relying only on `isCancelled`, tests were flaky. I'll check if this is still the case.